rest/route/auth.go

```diff
@@ -36,6 +37,11 @@ func (h *authPermissionGetHandler) Parse(ctx context.Context, r *http.Request) e
 func (h *authPermissionGetHandler) Run(ctx context.Context) gimlet.Responder {
 	u := MustHaveUser(ctx)
 
+	// TODO: remove with PM-1355
+	if !evergreen.AclCheckingIsEnabled {
+		return gimlet.NewTextResponse(false)
```

Comment:
this looks wrong

Reply:
that is because it is :)

Reply:
Don't you just want to return 200 (gimlet.NewJSONResponse(struct{}{}), and bedone. if there's checkuser that passes here, then allow it until acl is turned on.